test/application_system_test_case.rb

```diff
@@ -0,0 +1,19 @@
+require "test_helper"
+require "webdrivers"
+
+Capybara.register_driver :headless_chrome do |app|
+  options = ::Selenium::WebDriver::Chrome::Options.new
+  options.add_argument("--headless") unless ENV["HEADFUL"].in?(%w[y t 1 yes true])
```

Comment:
Can we use `ActiveModel::Type::Boolean.new.cast(ENV["HEADFUL"])` here?

Reply:
Yep. Done!